app/src/router.js

```diff
@@ -0,0 +1,37 @@
+import Vue from 'vue';
+import Router from 'vue-router';
+
+Vue.use(Router);
+
+const router = new Router({
+  mode: 'history',
+  base: process.env.BASE_URL,
+  routes: [
+    {
+      path: '/',
+      name: 'welcome',
+      component: () => import(/* webpackChunkName: "welcome" */ './views/Welcome.vue'),
```

Comment:
this is a small enough app that it's likely easier to keep it as a single chunk, rather than trying to do async / deferred loading